drivers/dma/dma_mcux_lpc.c

```diff
@@ -41,6 +41,8 @@ struct channel_data {
 	void *user_data;
 	dma_callback_t dma_callback;
 	enum dma_channel_direction dir;
+	uint8_t src_inc;
```

Comment:
We should clear these values in the `dma_mcux_lpc_clear_channel_data` function

Reply:
Added this code.